docs/styles/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Roboto", Arial, sans-serif;
+  line-height: 1.5; /* baseline height 24p x*/
+}
+
+p {
+  font-size: 1rem;
+  margin-bottom: 1.5rem;
+}
+
+p.lead {
+  font-size: 1.2rem;
+  line-height: 1.5rem;
```

Comment:
I would recommend you to use line-height in general styling (headings, paragraphs, …) always as multiples (like you did in the `body`). This saves you from adjusting it every single time you do adjustments on the font-size. multiples means something like `1.5` then this value gets calculated like: `current font-size × 1.5`. Check also the other selectors for that.